include/accessor.h

```diff
@@ -261,44 +286,81 @@ class accessor<DataT, Dims, Mode, target::device> : public detail::accessor_base
 	}
 
 	template <typename Functor>
-	accessor(const ctor_internal_tag /* tag */, const buffer<DataT, Dims>& buff, handler& cgh, Functor rmfn) {
+	accessor(const ctor_internal_tag /* tag */, const buffer<DataT, Dims>& buff, handler& cgh, const Functor& rmfn) {}
```

Comment:
Yes, same thing for `host_task` accessors!

Reply:
I have moved the device-accessor constructor body back into the `internal_tag`ged overload. Since there's no `ctor_internal_tag` overloads for `host_task`, I'll leave that one as is.